autoload/neomake/makers/ft/d.vim

```diff
@@ -0,0 +1,55 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#d#EnabledMakers()
+    "dmd, ldmd, and gdmd all share a common CLI.
+    "Ordered in efficiency of compiler
+    let l:makers = []
+    if neomake#utils#Exists('dmd')
```

Comment:
Wouldn't all three run on a make then?
They all use the same frontend code so it offers no benefit to run more than one.